scripts/build/termux_step_massage.sh

```diff
@@ -97,11 +111,29 @@ termux_step_massage() {
 	local HARDLINKS
 	HARDLINKS="$(find . -type f -links +1)"
 	if [ -n "$HARDLINKS" ]; then
-		termux_error_exit "Package contains hard links: $HARDLINKS"
+		if [ "$TERMUX_PACKAGE_LIBRARY" = "bionic" ]; then
+			termux_error_exit "Package contains hard links: $HARDLINKS"
+		elif [ "$TERMUX_PACKAGE_LIBRARY" = "glibc" ]; then
+			local declare hard_list
+			for i in $HARDLINKS; do
+				hard_list[$(ls -i "$i" | awk '{printf $1}')]+="$i "
+			done
+			local root_file
+			for i in ${!hard_list[@]}; do
+				root_file=""
+				for j in ${hard_list[$i]}; do
+					if [ -z "$root_file" ]; then
+						root_file="$j"
+						continue
+					fi
+					ln -sf "${TERMUX_PREFIX_CLASSICAL}/${root_file:2}" "${j}"
```

Comment:
> Why this approach?

Because I don't know how to properly force the compiler to use symbolic links instead of hard ones. Of course, I can use patches, but that sounds like a so-so idea.

> What happens if three (or more) files are hardlinks?

This algorithm assumes such cases, so it will work. But if you briefly explain, then the first hardlink remains as the original and all the rest become symbolic links to that original.